src/mon/MonClient.h

```diff
@@ -213,14 +213,19 @@ class MonClient : public Dispatcher {
   void handle_subscribe_ack(MMonSubscribeAck* m);
 
   bool _sub_want(string what, version_t start, unsigned flags) {
-    if ((sub_new.count(what) == 0 &&
-	 sub_sent.count(what) &&
-	 sub_sent[what].start == start &&
-	 sub_sent[what].flags == flags) ||
-	(sub_new.count(what) &&
-	 sub_new[what].start == start &&
-	 sub_new[what].flags == flags))
+    auto sub_new_iter = sub_new.find(what);
```

Comment:
@tchaikov 
What's wrong with adding iter to name of the container? Doesn't it make very explicit what variable is doing?

If I rename sub_new_iter to sub, how should I name sub_sent_iter?